presto-main/src/main/java/io/prestosql/spiller/NodeSpillConfig.java

```diff
@@ -89,4 +91,16 @@ public NodeSpillConfig setSpillDirectSerdeEnabled(boolean spillDirectSerdeEnable
         this.spillDirectSerdeEnabled = spillDirectSerdeEnabled;
         return this;
     }
+
+    public int getSpillPrefetchReadPages()
+    {
+        return spillPrefetchReadPages;
+    }
+
+    @Config("experimental.spill-prefetch-read-pages")
```

Comment:
Dont we need to have a Max config? else the configuration can be made to peform DOS attack

Reply:
Added Max config